### DISCUSSION TITLE
a "virgin mode" on r-link2 problem

### DISCUSSION THREAD

**Discussion by nathnd15** [cedricp/ddt4all#1138] [Q&A]
hello, I need help because I accidentally pressed the "virgin mode" option and I don't know how to get out of it. This is urgent so thank you for your response.

**Comment by Furtif** [comment]
look https://github.com/cedricp/ddt4all/discussions/1133

**Reply by nathnd15** [reply]
but my renault espace v screen is black

**Reply by nathnd15** [reply]
and he did nothing

**Reply by nathnd15** [reply]
je suis allé dans le mode developper et j'ai appuyé sur la fonction virgin mode depuis, j'ai l'impression que mon écran central est débranché voire mort

**Reply by Furtif** [reply]
il n'est pas mort, tu as simplement formaté le post audio en cliquant sur 'make virgin ' maintenant il te faut une cle usb avec la version du Os du post et brancher un diag tool pour creer et configurer la base ecu audio unit, pour cela je te demander de lire les post que parlent de make virgin, connect diag tool, etc.

**Reply by Furtif** [reply]
tu peut rejoindre le group facebook spécialisé dans le domaine https://www.facebook.com/groups/140606960090892 (francophone) ![image](https://github.com/cedricp/ddt4all/assets/11375035/ccebe3f3-797d-4fd5-bfb2-7898c911f325)

Ou tenter toi même réparer la bourde.

**Reply by nathnd15** [reply]
> il n'est pas mort, tu as simplement formaté le post audio en cliquant sur 'make virgin ' maintenant il te faut une clé usb avec la version du Os du post et brancher un outil de diagnostic pour créer et configurer la base ecu audio unit, pour cela je te demande de lire les post qui parlent de make virgin, connect diag tool, etc.

sur le groupe facebook ou sur une autre discussion github?

**Reply by nathnd15** [reply]
> tu peux rejoindre le groupe facebook spécialisé dans le domaine https://www.facebook.com/groups/140606960090892 (francophone)![image](https://private-user-images.githubusercontent.com/11375035/337871141-ccebe3f3-797d-4fd5-bfb2-7898c911f325.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc4NDc0MjIsIm5iZiI6MTcxNzg0NzEyMiwicGF0aCI6Ii8xMTM3NTAzNS8zMzc4NzExNDEtY2NlYmUzZjMtNzk3ZC00ZmQ1LWJmYjItNzg5OGM5MTFmMzI1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA4VDExNDUyMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFlNTViNWE3Nzc5ODM3NDY4OWVkNjI4MjFlMzE2ZDNjNmYyY2RmNTJlZjc5YjkzZTQ2MjJmYjQ0MDE3NmJlZGUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.mIxEx3XvvmKQ2idDz38RIJjzy50mie3NlrYmgRkpHcA)
> 
> Ou tenter toi même réparer la bourde.

haha je vais faire confiance aux specialistes car j'ai bien vu que j'étais nul

**Reply by nathnd15** [reply]
d'ailleurs ma demande d'adhésion est en attente et je ne sais pas combien de temps celle-ci mettra afin d'être acceptée

**Reply by Furtif** [reply]
> > il n'est pas mort, tu as simplement formaté le post audio en cliquant sur 'make virgin ' maintenant il te faut une clé usb avec la version du Os du post et brancher un outil de diagnostic pour créer et configurer la base ecu audio unit, pour cela je te demande de lire les post qui parlent de make virgin, connect diag tool, etc.
> 
> sur le groupe facebook ou sur une autre discussion github?

Tu peut voir les 2, le savoir n'occupe pas de la place... (il ne serait pas aussi dangereux lire les 2 comme appuyer sur make a virgin).

**Reply by Furtif** [reply]
> d'ailleurs ma demande d'adhésion est en attente et je ne sais pas combien de temps celle-ci mettra afin d'être acceptée

cela ne dépends pas de moi, malgré que je suis dans ce groupe ne ne suis pas modérateur, ou admin.

**Reply by nathnd15** [reply]
mais si je comprend bien le probleme enfaite il faut que je fasse un diagnostique de ma voiture et apres je dois executer un code python?

**Reply by Furtif** [reply]
en gros tu a appuyer sur ```xxx virgin``` surement dans micom test par le coup ce bouton formate le poste ou par d'autres mots le rends vierge sans aucun donné, par le fait le post reste 'noir' et/ou demandera par la suite 'System not found, please connect diag tool' ou autre chose, dépendent du modèle de chaque truc. Comme il na pas de system ou partie ECU configuré y a problème:
#### Etapes:
 - Connecter le diag tool (DDT4All ou autres)
 - Configurer l'ecu qui va avec (votre cas r-link2, théoriquement c'est un ecu UCL)
 - Introduire la clé usb réinstaller le OS (Operating system)
 - Entrer le code antivol du post
 - Prêt 
 
Mais bon... il ne convient pas seulement a faire tourner un code python, il tourne tout seul, visiter [Releases](https://github.com/cedricp/ddt4all/releases) et choisir.

Il vous faudra un certain nombre d'autres choses, comme [adapteurs ODB2](https://github.com/cedricp/ddt4all?tab=readme-ov-file#supported-diagnostic-adapters-so-far) que fassent l'affaire, pour d'autre cote faudra peut-être aussi faire quelque changements au niveaux de pin [(changer pins ou broches)](https://github.com/cedricp/ddt4all/discussions/970#discussioncomment-7917859) ODB a fin de pouvoir rentrer en mode CAN2 ou multimédia

**Reply by Furtif** [reply]
sinon méthode facile tu parles avec Jeremy Dacia sur facebook il te règle le bug en moins de deux.

**Reply by Furtif** [reply]
et maintenant tu le sais .... il ne faut pas touch a **make virgin**

**Reply by nathnd15** [reply]
merci beaucoup mais je comprend mal en faite je dois installer ddt4all sur une clé usb et la brancher sur mon écran central?

**Reply by Furtif** [reply]
tu dois, soit faire un sort que tu le saches refaire a la prochaine fois, (car rare ceux que on taper sur se buton le feront la prochaine fois) … soit tu vas sur facebook parle a Jeremy il te fais un miracle, après je ne te conseille rien, (n'oublie pas que tas fait de la merde juste dans micom test), mais bon… Il y a l'audacieux, et le me fieux … A toi de voir.
